src/components/organisms/FilmCard/filmCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { withRouter } from "react-router-dom";
+// import "./style.scss"
+
+import { 
+    Card,
+    Button,
+} from "react-bootstrap";
+
+import Sublabel from "../../molecules/Sublabel/sublabel"
+
+const FilmCard = (props) => {
+    const handleClick = (movie, poster) => {
```

Comment:
find another meaningful method name.

you invoke this on the `See detail` button but it doesn't tell the reader what it about to do?

like when your onClick --> handleClick. what does it do?
example more meaningful name:

`seeMovieDetail`. so when other developers see thiis line:
`<Button variant="primary" onClick={() => seeMovieDetail(props.movie, props.moviePoster)}>See detail</Button>`
she/he can understand more better what will happen the button clicked